jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/GoTimeUtilTest.java

```diff
@@ -32,8 +33,10 @@ class GoTimeUtilTest {
     @ParameterizedTest(name = "duration ''{0}'' should be ''{1}'' seconds")
     @MethodSource("data")
     void conversion(String duration, int expectedDuration) {
-        int result = GoTimeUtil.durationSeconds(duration).get();
-        assertThat(result).isEqualTo(expectedDuration);
+        Optional<Integer> optionalResult = GoTimeUtil.durationSeconds(duration);
+        assertThat(optionalResult)
```

Comment:
Could you please move the method call inline?

Reply:
void conversion(String duration, int expectedDuration) {
    assertThat(GoTimeUtil.durationSeconds(duration))
        .isPresent()
        .contains(expectedDuration);
}

will that work?

Reply:
Sure. You can move ahead with the review.